### ISSUE TITLE
Configuration of New Vaults for Spacewalk on Amplitude

### ISSUE THREAD

**Issue by annatekl** [pendulum-chain/spacewalk#449]
This ticket is for setting up new vaults on Amplitude to support the seamless bridging assets on Spacewalk.

**Comment by annatekl** [comment]
@ebma can you provide technical details?

**Comment by ebma** [comment]
Yes. But can you please specify in the tickets how many vaults you want to have and for which assets?

**Comment by annatekl** [comment]
@ebma I am talking to Bahila since she knows the process how to get in touch with Vaults, for now we don't know the details

**Comment by ebma** [comment]
@annatekl we are not dependent on external vault operators here as this is about internal vaults. I see that @vadaynujra updated the description. Thanks for that. Can you please also mention how much collateral you'd like these vaults to deposit? Tech team can then go ahead and create the accounts and prepare the commands for running them. Topping them up with the necessary KSM amounts would then be up to the product team again, but this has to wait until the accounts are created.

**Comment by annatekl** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/spacewalk/449?planning-poker) with Zenhub @b-yap @bogdanS98 @ebma @gianfra-t @TorstenStueber

**Comment by ebma** [comment]
Also here, let's only mention the new vaults and not the existing ones in order not to create confusion. And move it to the tasks repository.
